mod_auth/forms.py

```diff
@@ -131,12 +131,12 @@ def validate_password_repeat(form, field):
             raise ValidationError('The password needs to match the new password')
 
 
-class AccountForm(Form):
+class AccountForm(FlaskForm):
     """
     Form for editing current Account
     """
     def __init__(self, formdata=None, obj=None, prefix='', *args, **kwargs):
-        super(AccountForm, self).__init__(formdata, obj, prefix, *args, **kwargs)
+        super(AccountForm, self).__init__(*args, **kwargs)
```

Comment:
Did FlaskForm drop the extra arguments?

Reply:
It caused the problem
TypeError: __init__() takes from 1 to 2 positional arguments but 4 were given